src/assets/i18n/en_US.json

```diff
@@ -884,9 +884,37 @@
   "product.ready_for_shipment.within24": "Usually ready for shipment within 24 hours",
   "product.remove_all_product_filters.text": "Reset all filters",
   "product.retailSet.products.heading": "This retail set contains",
+  "product.review.content.error.required": "Please enter a review content.",
```

Comment:
Maybe better: "Please enter a comment." 
Else remove the "a" and change to "Please enter review content." But I actually prefer the other suggestion. ;-)

Reply:
Maybe better: "Please enter a review."